avm/res/network/route-table/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -0,0 +1,84 @@
+targetScope = 'subscription'
+
+metadata name = 'Using large parameter set'
+metadata description = 'This instance deploys the module with most of its features enabled.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-network.routetables-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'nrtcom'
```

Comment:
```suggestion
param serviceShort string = 'nrtwaf'
```

Reply:
Please update your fork with latest upstream and re-run the static tests. This should have been caught.

Before you do - I recommend temporarily disabling the actions though as you don't want to tigger all pipelines ;)